recipes/julia/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "julia" %}
+{% set version = "0.5.2" %}
+{% set sha256 = "efa3303d63e39bf3ea06e23ef68807cca0095c7e0507bdf26eae1d5dee7868c7" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [osx or win]
+  number: 0
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - gcc  # [linux]
+    - toolchain
+    - patchelf
+    - cmake
+    - zlib 1.2.8
+    - python 2.7.*
+    - autoconf 2.69 pl5.*
+    - automake 1.15 pl5.*
```

Comment:
what were you needing autotools for? likewise with libxml2, don't usually expect that to be a build-dep